src/Servers/Kestrel/test/BindTests/AddressRegistrationTests.cs

```diff
@@ -108,7 +108,6 @@ public async Task RegisterIPEndPoint_DynamicPort_Success(IPEndPoint endPoint, st
         [ConditionalTheory]
         [MemberData(nameof(IPEndPointRegistrationDataPort443))]
         [IPv6SupportedCondition]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore-internal/issues/2711")]
```

Comment:
Revert, the default port binding tests are inherently flaky and should remain in quarantine.